### DISCUSSION TITLE
StepsViewModel

### DISCUSSION THREAD

**Discussion by WasifRaza123** [brittanyarima/Steps#72] [General]
Hi @brittanyarima, 
In stepsViewModel, on [line](https://github.com/brittanyarima/Steps/blob/1c9a0bdff31aa9166a30a29bd62edde2dd402d74/Steps/ViewModel/StepsViewModel.swift#L26) the goal is set to 0 when the user has not set any goal but it should be set to 10000 as the default value. I think that this is a redundant line as the goal value will change when the user changes it as goal is of type @published.

I didn't know of any other way to communicate about the bug so I posted here.

```
self.goal = UserDefaults.standard.integer(forKey: Constants.goalKey)
```

**Comment by brittanyarima** [comment]
@WasifRaza123 oh good point! Thanks for pointing that out!
